File: 1672_RichestCustomerWealth.js
/**
 * @param {number[][]} accounts
 * @return {number}
 */
var maximumWealth = function(accounts) {
    var mapToSum = accounts.map(x=>x.reduce((a, b)=> a + b));
    var maxWealth = Math.max(...mapToSum);
    return maxWealth;
};

File: 1672_RichestCustomerWealth.java
class Solution {
    public int maximumWealth(int[][] accounts) {
        int maxWealth = 0;
        for(int i= 0; i < accounts.length; i++){
            int wealth = Arrays.stream(accounts[i]).sum();
            if (wealth > maxWealth){
                maxWealth = wealth;
            }
        }
        return maxWealth;
    }
}